src/lib/viewport-uniforms.js

```diff
@@ -155,6 +155,8 @@ export function getUniformsFromViewport(viewport, {
     modelViewMatrix: new Float32Array(modelViewMatrix),
 
     // Screen size
+    // _Q2# : Verify and remove multiplicaiton with devicePixelRatio (assuming
```

Comment:
@ibgreen @Pessimistress , is multiplication with `devicePixelRation` needed here? Who sets these viewport bounds?